aicoe/sesheta/review_manager.py

```diff
@@ -178,7 +178,7 @@ async def on_pr_open_or_edit(*, action, number, pull_request, repository, sender
 
             _LOGGER.debug(f"on_pr_open_or_edit: automatic update, will auto-approve {pull_request['html_url']}!")
 
-            if pull_request["organization"]["login"] == "thoth-station":
```

Comment:
@harshad16 pls have a look here

Reply:
https://docs.github.com/en/developers/webhooks-and-events/webhook-events-and-payloads#pull_request